python_modules/dagster/dagster/_grpc/client.py

```diff
@@ -376,18 +382,31 @@ def external_schedule_execution(
             ExternalScheduleExecutionArgs,
         )
 
-        chunks = list(
-            self._streaming_query(
-                "ExternalScheduleExecution",
+        try:
+            return self._query(
+                "SyncExternalScheduleExecution",
                 api_pb2.ExternalScheduleExecutionRequest,
                 serialized_external_schedule_execution_args=serialize_value(
                     external_schedule_execution_args
                 ),
                 timeout=timeout,
-            )
-        )
-
-        return "".join([chunk.serialized_chunk for chunk in chunks])
+            ).serialized_schedule_result
+        except Exception as e:
+            # On older servers that only have the streaming API call implemented, fall back to that API
+            if self._is_unimplemented_error(e):
+                chunks = list(
+                    self._streaming_query(
+                        "ExternalScheduleExecution",
+                        api_pb2.ExternalScheduleExecutionRequest,
+                        serialized_external_schedule_execution_args=serialize_value(
+                            external_schedule_execution_args
+                        ),
+                        timeout=timeout,
+                    )
+                )
+                return "".join([chunk.serialized_chunk for chunk in chunks])
```

Comment:
nit: unclear how much this is worth worrying about, but if we eat an exception while in this `except` block we'll get some additional exception noise that maybe advocates for trying to do this work outside of the `except` block? 

Eh I dunno, its probably fine.